1001-2000/LIT1259Daniel.xml

```diff
@@ -34,6 +34,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract>
+            <p>Generic ID for the biblical Book of Daniel, referring to a recension that does not include the stories of Susanna, Bel and the 
+               Dragon, the Prayer of Azariah and the Three Holy Children, which belong to the so-called deuterocanonical books, or to a or to a 
```

Comment:
or to a or to a double :)

Reply:
I think it is also useful to mark up the names of works in abstracts, so that users can find the corresponding page immediately without searching

Reply:
Do you think, I should point to LIT3529Daniel in this abstract or to a wikidata (Q80115)?

Reply:
I am not sure if the wikidata reference is resolved (if not, it is not necessarily needed, the references to the other Bm entries are enough, I think)